CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Fix the git input parameter `ref` to align with the `git` notion of a ref. This allows for the use 
   of branch names, tag names, and commit hashes.
+- Fix bug to check the directory of target proto file paths in fallback logic when scanning
```

Comment:
This says what the logic is doing, not what the bug was and what the user impact of it was. I'd expect something more along the lines of "Fix issue where [whatever was happening in 3183]", does that make sense? Also complete sentences (ie period at end)